architecture/spmc.rst

```diff
@@ -0,0 +1,269 @@
+====
+SPMC
+====
+
+This document describes the SPMC (S-EL1) implementation for OP-TEE.
+More information on the SPMC can be found in the FF-A specification can be
+found in the
+`FF-A spec <https://developer.arm.com/documentation/den0077/latest>`_.
+
+.. toctree::
+ :numbered:
+
+SPMC Responsibilities
+=====================
+
+The SPMC is a critical component in the FF-A flow. Some of its mayor
+responsibilities are:
+
+- Initialisation and run-time management of the SPs:
+	The SPMC component is responsible for initialisation of the
+	Secure Partitions (Loading the Image, Setting up the stack, HEAP, ...).
```

Comment:
- secure partitions (no initial capitalization)? This is arguable, like normal world / Normal World. What is the convention used in Arm documentation?
- loading, image, setting, heap

Reply:
The spec uses Secure Partitions